src/graph_service/README.md

```diff
@@ -0,0 +1,79 @@
+# Rampart Backend Service
+
+## Overview
+
+The Rampart Backend Service provides a backend for the Rampart UI to interact with Kubernetes. The Rampart UI has the concept of `App`s, which are Rampart graphs with version history (including with the ability to revert, etc). Unfortunately, this abstraction does not match the abstraction maintained by `RampartGraph` custom resources, which do not maintain any sort of versioning. Therefore, the Rampart Backend Service needs to store the mapping between `App`s and `RampartGraph` custom resources.
```

Comment:
Rampart UI does not interact with Kubernetes directly. I think it is better to say something like "... backend for Rampart UI to Interactively build rampart graphs and manage graph deployments."

Reply:
Updated